src/asm3/wordprocessor.py

```diff
@@ -153,6 +153,9 @@ def additional_field_tags(dbo, fields, prefix = ""):
             val = af["ANIMALNAME"]
         if af["FIELDTYPE"] in human_flags:
             val = af["OWNERNAME"]
+            human = asm3.additional.get_human_record(dbo, int(af["VALUE"]))[0]
+            for data in human:
+                tags[af["FIELDNAME"].upper() + data] = human[data]
```

Comment:
This appears to be adding every single column in the owner row as a tag? This is unnecessary and will use a lot of memory.

Instead, add a function where you can pass a person record, tags collection and the prefix (eg: additional_field_person_tags(p, tags, prefix). This function then adds specific named fields to the tags from the person record, probably the same tags as we do for CURRENTOWNER, ORIGINALOWNER, etc.

Reply:
I did do that actually ... that loop was only for testing.
accidently wrong PR (even if I checked before)